src/dialects/abstract/query-generator.js

```diff
@@ -47,12 +47,24 @@ export class AbstractQueryGenerator {
     this._initQuoteIdentifier();
   }
 
+  defaultSchema(dialect = this.options.dialect) {
```

Comment:
That is a great suggestion!

I looked at placing it elsewhere, but I'd need assistance. This requires a reference to the Sequelize object to access the connected config to retrieve username/databaseName details.  I'm not sure how that would be done.

Since Sequelize is not imported in the other places, I thought it could be an additional (unnecessary) performance hit to load/instantiate another library there.  I may be wrong on that front, obviously I need help 😄

Reply:
You can access the dialect through `this._dialect`. It's already present so no extra performance hit there.

The dialect also already has access to the sequelize instance (example, db2: https://github.com/sequelize/sequelize/blob/main/src/dialects/db2/index.js#L32). So a `getDefaultSchema` method in these classes wouldn't have any perf impact either